### ISSUE TITLE
Drag and drop inside subviewport not working.

### ISSUE THREAD

**Issue by Sp3ci3s8472** [godotengine/godot#78441]
### Godot version

4.0.2

### System information

Windows 10/EndeavourOS (Arch Linux)

### Issue description

I made some code to drag and drop inside a hboxcontainer. This seems to work fine when I use this in the main viewport. However, when I add this to a subviewport I can start the drag operation, but I can't confirm for a drop. Cancelling the drag and the drop operation works as expected.

I run the .net version of Godot; but I don't think the issue is language related.

To clarify, this worked in 3.*.

### Steps to reproduce

See the minimal reproduction project. The drag and drop in the main viewport works and the it fails in the greenish subviewport.

I will see if I have the time to create a GDScript, specific version if necessary. 

### Minimal reproduction project

[DragAndDrop.zip](https://github.com/godotengine/godot/files/11790299/DragAndDrop.zip)

**Comment by Sp3ci3s8472** [comment]
> * Duplicate of: [In-game Drag and Drop between viewports does not work #28522](https://github.com/godotengine/godot/issues/28522)

No, this worked in 3.* for me. Only after migrating I see this behaviour. I added this to the description now.

**Comment by Sp3ci3s8472** [comment]
> #67531?

Not precisely, but it might be related. I don't want to drop between the main vieport and subviewports.

**Comment by Sauermann** [comment]
I have installed .net, recompiled Godot with #67531 and can confirm, that it fixes the problem of this MRP. Feel free to close this issue.

**Comment by akien-mga** [comment]
- Duplicate of https://github.com/godotengine/godot/issues/28522
